doc/classes/Image.xml

```diff
@@ -351,29 +376,43 @@
 			<argument index="2" name="color" type="Color">
 			</argument>
 			<description>
+				Sets the [Color] of the pixel at [code](x, y)[/code] if the image is unlocked. Example:
+				[code]
+				var img = Image.new()
+				img.lock()
+				img.set_pixel(x, y, color) # Does not have an effect
+				img.unlock()
+				img.set_pixel(x, y, color) # Works
+				[/code].
 			</description>
 		</method>
 		<method name="shrink_x2">
 			<return type="void">
 			</return>
 			<description>
+				Shrinks the image by a factor of 2.
 			</description>
 		</method>
 		<method name="srgb_to_linear">
 			<return type="void">
 			</return>
 			<description>
+				Converts the raw data from the sRGB colorspace to a linear scale.
 			</description>
 		</method>
 		<method name="unlock">
 			<return type="void">
 			</return>
 			<description>
+				Unlocks the data for writing access.
 			</description>
 		</method>
 	</methods>
 	<members>
-		<member name="data" type="Dictionary" setter="_set_data" getter="_get_data">
+		<member name="data" type="Dictionary">
+			<description>
+				Holds all of the image's color data in a given format. See [code]FORMAT_*[/code] constants.
+			</description>
```

Comment:
- You removed the setter and getter. This caused #12466, although now that I think of it, I shouldn't relay on the docs for this. This made me find another bug: `doctool` doesn't fix setters and getters. If the member in the xml has none, it generates `setter="" getter=""`; if I change the setter/getter to a wrong one, it leaves it as is without fixing.
- Members don't use the `description` node. You place the description in the member itself (or that's what I see in other files.., no idea). Because of this, the next time you run `doctool` the description will be removed.

ping @godotengine/documentation

Reply:
Fixed.

Reply:
> This made me find another bug: doctool doesn't fix setters and getters. If the member in the xml has none, it generates setter="" getter=""; if I change the setter/getter to a wrong one, it leaves it as is without fixing.

I've just noticed it too and was about to report it :) Well, I will.

Reply:
I've opened #12476 for that.